### PR TITLE
Eks

### PR DESCRIPTION
**Pull request by MichalFupso**

### CODE REVIEW

File: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx [hunk 1]
```diff
@@ -113,12 +113,34 @@ Before you get started, make sure you have downloaded and configured the [necess
    kubectl apply -f {{manifestsUrl}}/manifests/calico-vxlan.yaml
    ```
 
-1. Configure {{prodname}} to disable AWS src/dst checks.
+2. Configure {{prodname}} to disable AWS src/dst checks.
```
Comment (mgleung) [inline calico/getting-started/kubernetes/managed-public-cloud/eks.mdx]: Do the numbers here need to be changed manually? I thought we always put `1` and liquid handles the explicit numbering for us?

File: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx [hunk 2]
```diff
@@ -113,12 +113,34 @@ Before you get started, make sure you have downloaded and configured the [necess
...
 
    ```batch
    kubectl -n kube-system set env daemonset/calico-node FELIX_AWSSRCDSTCHECK=Disable
    ```
 
+</TabItem>
+
+<TabItem label="Helm" value="Helm-2">
```
Comment (mgleung) [inline calico/getting-started/kubernetes/managed-public-cloud/eks.mdx:135]: I might be misunderstanding what this does, but I'm kind of confused why we have tabs here for helm when I don't see any other tabs. Are there other tabs that I'm missing?

Comment (mgleung) [reply calico/getting-started/kubernetes/managed-public-cloud/eks.mdx:135]: Nvm, I see that I misread things. This should be fine.

**Review by mgleung — DISMISSED** [review]

File: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico/getting-started/kubernetes/managed-public-cloud/eks.mdx:135]: Although already introduced by previous author, using inline tabs inside a procedure has issues. The final step for all tabs comes out as "1." in netlify. I see two options. Duplicate the content for each tab (which is the original intent of tabs), or make the final step to be a non-numbered step. Like this:

Finally, add nodes to the cluster.

image

File: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx [hunk 3]
```diff
@@ -119,6 +119,28 @@ Before you get started, make sure you have downloaded and configured the [necess
...
+
+1. Add {{prodname}} into your Helm repository.
+
+    ```batch
+    helm repo add projectcalico https://docs.projectcalico.org/charts
+    ```
+
+1. If you already have {{prodname}} added, you can update it to get the latest released version.
```
Comment (bmckercher123) [inline calico/getting-started/kubernetes/managed-public-cloud/eks.mdx]:
```suggestion
1. If {{prodname}} is already added, update it to get the latest released version.
```

**Review by bmckercher123 — DISMISSED** [review]
Minor nits.

**Review by bmckercher123 — APPROVED** [review]
LGTM

File: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx (hunk 1 above)
Comment (ctauchen) [reply calico/getting-started/kubernetes/managed-public-cloud/eks.mdx]: There are circumstances where they need to be explicit, yes. I'm not sure of the exact cause. But having multiple paragraphs for a single step, particularly with different markup, such as that for notes and codeblocks, significantly increases the likelihood that automatic numbering will fail.

**Comment by ctauchen** [comment]
@MichalFupso If all comments have been addressed, then we're about ready to merge this one. 

Could you please squash your commits and rebase?

**Pull request merged**